sd-ran/values.yaml

```diff
@@ -0,0 +1,155 @@
+    enabled: true
+  onos-ric:
+    enabled: true
+  onos-ric-ho:
+    enabled: true
+  onos-ric-mlb:
+    enabled: true
+  onos-topo:
+    enabled: true
+  onos-gui:
+    enabled: true
+  ran-simulator:
+    enabled: true
+  onos-cli:
+    enabled: true
+
+# NEM-Monitoring
+nem-monitoring:
+  prometheus:
+    serverFiles:
+      prometheus.yml:
+        scrape_configs:
+          - job_name: 'onos-ric'
+            metrics_path: /metrics
+            scrape_interval: 1s
+            static_configs:
+              - targets:
+                  - onos-ric.micro-onos.svc.cluster.local:7000
+          - job_name: 'onos-ric-ho'
+            metrics_path: /metrics
+            scrape_interval: 1s
+            static_configs:
+              - targets:
+                  - onos-ric-ho.micro-onos.svc.cluster.local:7001
+          - job_name: 'onos-ric-mlb'
+            metrics_path: /metrics
+            scrape_interval: 1s
+            static_configs:
+              - targets:
+                  - onos-ric-mlb.micro-onos.svc.cluster.local:7002
+          - job_name: 'ran-simulator'
+            metrics_path: /metrics
+            scrape_interval: 1s
+            static_configs:
+              - targets:
+                  - ran-simulator.micro-onos.svc.cluster.local:9090
+  dashboards:
+    nodeExporter: false
+  kpi_exporter: 
+    enabled: false
+  grafana:
+    datasources:
+      datasources.yaml:
+        apiVersion: 1
+        datasources:
+          - name: Prometheus
+            type: prometheus
+            url: http://sd-ran-prometheus-server
+            access: proxy
+            isDefault: true
+
+# ONOS-RIC
+onos-ric:
+  image:
+    repository: onosproject/onos-ric
+    tag: latest
+    pullPolicy: IfNotPresent
+  debug: false
+  store:
+    controller: "atomix-controller.kube-system.svc.cluster.local:5679"
+    consensus:
+      enabled: true
+      database: ""
+      partitions: 1
+      clusters: 1
+      backend:
+        image: atomix/raft-replica:latest
+        replicas: 1
+
+# ONOS-RIC-HO
+onos-ric-ho:
+  image:
+    repository: onosproject/onos-ric-ho
+    tag: latest
+    pullPolicy: IfNotPresent
+
+# ONOS-RIC-MLB
+onos-ric-mlb:
+  image:
+    repository: onosproject/onos-ric-mlb
+    tag: latest
+    pullPolicy: IfNotPresent
+
+# ONOS-TOPO
+onos-topo:
+  image:
+    repository: onosproject/onos-topo
+    tag: latest
+    pullPolicy: IfNotPresent
+  store:
+    enabled: true
+    raftgroup: onos-topo-raft
+    controller: atomix-controller.micro-onos.svc.cluster.local:5679
+    raft:
+      partitions: 1
+      clusters: 1
+      backend:
+        image: atomix/raft-replica:latest
+        replicas: 1
+
+# ONOS-GUI
+onos-gui:
+  image:
+    repository: onosproject/onos-gui
+    tag: latest
+    pullPolicy: IfNotPresent
+
+# RAN-Simulator
+ran-simulator:
+  image:
+    repository: onosproject/ran-simulator
+    tag: latest
+    pullPolicy: IfNotPresent
+  fade: true
+  googleApiKey: "YOUR_API_KEY_HERE"
+  locationsScale: 1.0
```

Comment:
does this override the 1.25 from above?

Reply:
Likewise, 1.25 is a default value, I will modify it.

Reply:
Updated.